sdcm/nemesis.py

```diff
@@ -4126,6 +4127,44 @@ def disrupt_create_index(self):
             finally:
                 drop_index(session, ks, index_name)
 
+    def disrupt_add_remove_mv(self):
+        """
+        Create a Materialized view on an existing table while a node is down.
+        Take node up and run a repair.
+        Verify the MV can be used in a query.
+        Finally, drop the MV.
+        """
+        node1, node2 = self.cluster.nodes[:2]
+        self.log.info("Stopping Scylla on node1")
+        ks_cfs = self.cluster.get_non_system_ks_cf_list(db_node=node2, filter_empty_tables=True, filter_out_mv=True)
+        if not ks_cfs:
+            raise UnsupportedNemesis(
+                'Non-system keyspace and table are not found. nemesis can\'t be run')
+        ks_name, base_table_name = random.choice(ks_cfs).split('.')
+        view_name = f'{base_table_name}_view'
+        node1.stop_scylla()
+        with self.cluster.cql_connection_patient(node2) as session:
+            partition_key_name = get_partition_key_name(session=session, ks=ks_name, cf=base_table_name)
+            column = f'"{get_random_column_name(session=session, ks=ks_name, cf=base_table_name)}"'
+            InfoEvent(message=f'Create a materialized-view for table {ks_name}.{base_table_name}').publish()
+            try:
+                self.tester.create_materialized_view(ks_name, base_table_name, view_name, [column],
+                                                     [partition_key_name], session,
+                                                     mv_columns=[column, partition_key_name])
+            except Exception as error:  # pylint: disable=broad-except
+                self.log.warning('Failed creating a materialized view: %s', error)
+                return
```

Comment:
if we failed to create a MV, IMHO we shouldn't just `return`, but raise some error event (the nemesis will be marked as failed, but we need to indicate the most information we have why it happened)... even if later we decide to lower the severity